packages/trader/src/AppV2/Containers/Trade/trade-types.tsx

```diff
@@ -178,14 +179,26 @@ const TradeTypes = ({ contract_type, onTradeTypeSelect, trade_types }: TTradeTyp
                         <Text size='sm'>{title}</Text>
                     </Chip.Selectable>
                 ))}
+            {!saved_pinned_trade_types[0]?.items.some((item: TItem) => item.id === contract_type) &&
+                saved_other_trade_types.length > 0 &&
+                saved_other_trade_types[0].items
+                    .filter((item: TItem) => item.id === contract_type)
+                    .map(({ title, id }: TItem) => (
+                        <Chip.Selectable key={id} onChipSelect={onTradeTypeSelect} selected={isTradeTypeSelected(id)}>
+                            <Text size='sm'>{title}</Text>
+                        </Chip.Selectable>
+                    ))}
             <a key='trade-types-all' onClick={() => setIsOpen(true)} className='trade__trade-types-header'>
                 <Text size='sm' bold underlined>
                     {<Localize i18n_default_text='View all' />}
                 </Text>
             </a>
             <ActionSheet.Root isOpen={is_open} expandable={false} onClose={handleCloseTradeTypes}>
                 <ActionSheet.Portal>
-                    <ActionSheet.Header title={<Localize i18n_default_text='Trade types' />} />
+                    <ActionSheet.Header
+                        title={<Localize i18n_default_text='Trade types' />}
+                        icon={!is_editing && <Guide show_guide_for_selected_contract />}
```

Comment:
Flag show_guide_for_selected_contract  will show guide only for 1 trade type (currently selected). Do you want to achieve this or to show guide info for all trade types?

Reply:
I guess the other one